From 89c316ebf4ede98a282e98cb357cbafb261c70e0 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Mon, 2 Jun 2025 07:17:04 +0200
Subject: [PATCH] Revert "Make mold slightly happier."

This reverts commit 1c1bdb4f4aba988553a012c8cc0eb85b6e8aa662.

It broke build on my (musl-based) system; which keeps complaining about
a missing Kiface symbol in idf2vrml and dxf2idf, no matter which linker
is being used (tested with: BFD ld, gold, lld, and mold).

This also affects some QA tests.

Ref: https://github.com/rui314/mold/discussions/1234
Ref: https://gitlab.com/kicad/code/kicad/-/issues/20038
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 utils/idftools/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/utils/idftools/CMakeLists.txt b/utils/idftools/CMakeLists.txt
index 8a0cb4a8bf2d..64728d61eb45 100644
--- a/utils/idftools/CMakeLists.txt
+++ b/utils/idftools/CMakeLists.txt
@@ -31,7 +31,6 @@ target_link_libraries( dxf2idf
     dxflib_qcad
     idf3
     kimath
-    common
     ${OPENGL_LIBRARIES}
     ${wxWidgets_LIBRARIES}
     )
@@ -39,7 +38,6 @@ target_link_libraries( dxf2idf
 target_link_libraries( idf2vrml
     idf3
     kimath
-    common
     ${OPENGL_LIBRARIES}
     ${wxWidgets_LIBRARIES}
     )
-- 
2.50.1
